package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native",
-  "version": "0.12.0",
+  "version": "0.17.0-rc",
```

Comment:
Yes, but what should be the version number ? react native 0.16 is out but master version is 0.12.0